lektor_s3.py

```diff
@@ -33,6 +36,16 @@ def __init__(self, env, output_path):
         self.bucket = None
         self.key_prefix = ''
         self.cloudfront = None
+        self.config = self.get_config()
+
+    @property
+    def config_filename(self):
+        return os.path.join(self.env.root_path, 'configs', 's3.ini')
+
+    def get_config(self):
```

Comment:
I'm totally happy to use the `S3Plugin.get_config` method (that would also be my preference, assuming the API still allows me to accomplish the task) but I think I'm missing something as I don't see any way for the plugin to pass the config to the publisher. This is probably my lack of understanding as to how lektor plugins actually work, but it looks to me like `S3Plugin` just registers the `S3Publisher` with lektor and doesn't initialize it itself. I don't see any appropriate events listed here either: https://www.getlektor.com/docs/api/plugins/events/

Could you point me in the right direction for getting the config from the plugin to publisher?